### PR TITLE
[DOCS Add 'mostly' missing sub title in TOC

### PR DESCRIPTION
**Pull request by sephiartlist**

### CODE REVIEW

**Comment by sephiartlist** [comment]
@cla-bot check

**Review by talagluck — COMMENTED** [review]
Thanks so much for this contribution, @sephiartlist! I totally get wanting to add this here, though unlike the other four above, `mostly` is not shared by all expectations (e.g. aggregate expectations don't have a `mostly` argument), and so falls into its own category here. Any thoughts?

**Comment by sephiartlist** [comment]
We can add a clearer disclaimer 
> [mostly](#mostly): A special argument that allows for _fuzzy_ validation based on some percentage (available for all ONLY `column_map_expectations`)

**Comment by talagluck** [comment]
Since we say on L6 that these four attributes on shared by all Expectations, maybe we have another line before `mostly` saying something like "All `column_map_expectations` also have:" to  distinguish it a bit. What do you think?

**Comment by sephiartlist** [comment]
Sounds good by me - 
However I tried to find a similar situations in a different section in the docs - but could not find any.
what kind of line did you think of?

File: docs/reference/expectations/standard_arguments.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ All Expectations return a JSON-serializable dictionary when evaluated, and share
 * [catch_exceptions](#catch_exceptions): If true, execution will not fail if the Expectation encounters an error.
   Instead, it will return success = False and provide an informative error message.
 * [meta](#meta): Allows user-supplied meta-data to be stored with an Expectation.
+* [mostly](#mostly): A special argument that allows for _fuzzy_ validation based on some percentage (available for all `column_map_expectations`)
```
Comment (talagluck) [inline docs/reference/expectations/standard_arguments.md:15]:
```suggestion

All `ColumnMapExpectations` also have the following argument:
* [mostly](#mostly): A special argument that allows for _fuzzy_ validation based on some percentage (available for all `column_map_expectations`)
```

**Review by talagluck — COMMENTED** [review]
I think something like this suggestion should work

File: docs/reference/expectations/standard_arguments.md (hunk 1 above)
Comment (sephiartlist) [reply docs/reference/expectations/standard_arguments.md:15]: Good by me

**Review by talagluck — APPROVED** [review]
LGTM!

**Pull request merged**